src/FixedDataTableRoot-test.js

```diff
@@ -100,32 +105,65 @@ describe('FixedDataTableRoot', function() {
   });
 
   describe('update render', function() {
+    it('should update scrollLeft correctly', function() {
+      let table = renderTable({scrollLeft: 300});
+      assert.equal(table.getTableState().scrollX, 300, 'should set scrollX to 300');
+      table = renderTable({scrollLeft: 600});
+      assert.equal(table.getTableState().scrollX, 600, 'should set scrollX to 600');
+    });
+
+    it('should update scrollTop correctly', function() {
+      let table = renderTable({scrollTop: 600});
+      assert.equal(table.getTableState().scrollY, 600, 'should set scrollY to 600');
+
+      table = renderTable({scrollTop: 300});
+      assert.equal(table.getTableState().scrollY, 300, 'should set scrollY to 300');
+    });
 
+    it('should update scrollToColumn correctly', function() {
+      let table = renderTable({scrollToColumn: 3});
+      assert.equal(table.getTableState().scrollX, 300 * 2, 'should be third visible column');
```

Comment:
how does the mapping here work?  it's weird scrolling to the first is * 1 and scrolling to the 3rd is * 2.  Can you add a short comment on why that happens